Vagrantfile

```diff
@@ -8,12 +8,14 @@ gopath_folder="/opt/gopath"
 
 cluster_ip_nodes = ""
 
-provision_common_once = <<SCRIPT
+provision_common = <<SCRIPT
```

Comment:
Looks like the updated diff still has Vagrantfile changes. You might want to checkout the Vagrantfile from master.